api-client/src/routes/RouteInformationTypes.js

```diff
@@ -26,6 +27,11 @@ export type LandingRouteInformationType = {|
   languageCode: string
 |}
 
+export type JpalEvaluationRouteInformationType = {|
+  route: JpalEvaluationRouteType,
+  trackingCode: string | null
```

Comment:
In [IGAPP-527](https://issues.tuerantuer.org/browse/IGAPP-527) I decided it would make sense also to save if tracking is enabled/disabled. Would it be necessary to add this info to routeInfo or not?

Reply:
I don't think this should and can be part of the route information. A user has to consent to all terms and conditions, this is not possible as part of an url.